src/components/labeled-button.tsx

```diff
@@ -1,41 +1,32 @@
 import React from 'react';
-import { ButtonProps, Flex, ForwardRef, IconButton, Label } from 'theme-ui';
+import { Button, ButtonProps, ForwardRef, ThemeUIStyleObject } from 'theme-ui';
 
 export interface LabeledButtonProps extends ButtonProps {
-  label: string;
   name?: string;
```

Comment:
What is this field for?

Reply:
This was how it used to set the text on the button. The label prop would be put inside a `Label` element.

Reply:
This meant that we were unable to put the icon in front of the text. It was hard coded to put the label first, then the children elements.